agent/app/agent.go

```diff
@@ -61,6 +61,7 @@ import (
 	"github.com/aws/amazon-ecs-agent/ecs-agent/eventstream"
 	"github.com/aws/amazon-ecs-agent/ecs-agent/logger"
 	"github.com/aws/amazon-ecs-agent/ecs-agent/logger/field"
+	ecs_agent_metrics "github.com/aws/amazon-ecs-agent/ecs-agent/metrics"
```

Comment:
nit: import it as `metrics` looks better to me.

Reply:
There is an already imported metrics at line 46. Thats the reason I used another name to avoid confusion.